dcc-storage-client/src/main/java/org/icgc/dcc/storage/client/command/DownloadCommand.java

```diff
@@ -68,8 +68,8 @@
   private boolean force = false;
   @Parameter(names = "--manifest", description = "Path to manifest id, url or file")
   private ManifestResource manifestResource;
-  @Parameter(names = "--object-id", description = "Object id to download", validateValueWith = ObjectIdValidator.class)
-  private String objectId;
+  @Parameter(names = "--object-id", description = "Object id to download", validateValueWith = ObjectIdListValidator.class, variableArity = true)
```

Comment:
The Validator is looking to parse each value to make sure it's a valid UUID. The validation method is explicitly looking for a String, not an ArrayList